workspaces/linguist/plugins/linguist-backend/optic.yml

```diff
@@ -0,0 +1,16 @@
+ruleset:
+    - breaking-changes
+capture:
+    src/schema/openapi.yaml:
+        # 🔧 Runnable example with simple get requests.
+        # Run with "PORT=3000 optic capture src/schema/openapi.yaml --update interactive" in '/Users/bforbis/git/backstage-community-plugins/workspaces/linguist/plugins/linguist-backend'
+        # You can change the server and the 'requests' section to experiment
+        server:
+            # This will not be used by 'backstage-repo-tools schema openapi test', but may be useful for interactive updates.
+            url: http://localhost:3000
+        requests:
+            # ℹ️ Requests should be sent to the Optic proxy, the address of which is injected into 'run.command's env as OPTIC_PROXY (or the value of 'run.proxy_variable').
+            run:
+                # 🔧 Specify a command that will generate traffic
+                command: yarn backstage-cli package test --no-watch "src/service/router.test.ts" "src/service/createRouter.test.ts" 
+  
```

Comment:
I have to update the init command, but this is no longer necessary, you can run the tests in-band now, https://github.com/backstage/backstage/pull/25538

Reply:
Unless this was added to create the initial spec, then carry on 🫡

Reply:
I actually didn't look to closely at this file, I think this was added by the `generate` command. I see that it added a bunch of comments with absolute paths from my local file system, so at the very least I should clean those out.

So are you saying I should be able to delete the entire `optic.yml` file, or only the stuff under `capture`?